src/main/java/step3/domain/RacingCars.java

```diff
@@ -0,0 +1,33 @@
+package step3.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import step3.runType.NormalRunStrategy;
+import step3.runType.RunStrategy;
+
+public class RacingCars {
+
+    public static final String CAR_NUMBER_FORMAT = "car_number_%d";
+
+    private final List<Car> cars;
+
+    public RacingCars(int carCount) {
+        List<Car> preparedCars = prepareCars(carCount, new NormalRunStrategy());
+        this.cars = preparedCars;
```

Comment:
```suggestion
        this.cars = prepareCars(carCount, new NormalRunStrategy());
```
Cars의 Strategy가 생성자에서 고정시킨 상태라서 유연하지가 못하네요.
따라서 Strategy는 제일 최상위 레이어에서 만들어서 주입할 수 있도록 해주세요!

Reply:
답변감사합니다. 
'최상위 레이어'라 하심은 메인머서트가 있는 부분을 말씀하시는건가요? 
일단 메인메서드에서 Strategy를 주입하도록 수정했습니다.